web/themes/joinup/templates/join-collection-form.html.twig

```diff
@@ -0,0 +1,27 @@
+{#
+/**
+ * @file
+ * Template to render the share content form.
```

Comment:
Several remarks about this template:
- could be named better, as it's not for joining, but seems mainly concerned about leaving
- comment blocks are wrong
- not placed in correct folder